modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -770,6 +792,8 @@ export default {
       if (this.wizard.values.toLocalize.data === 'thisDoc') {
         return;
       }
+      const status = this.wizard.busy;
```

Comment:
Is this unused?

Reply:
Nope, it's smart enough to keep the previous busy state, see `this.wizard.busy = status;` at the end of the method